src/components/notes/saving-indicator.styl

```diff
@@ -0,0 +1,9 @@
+.saving-indicator
+    position absolute
+    bottom 1rem
+    right 1rem
+    padding 0.25rem 0.75rem
+    color var(--cool-grey, #95999d)
+    background-color: #FFFFFFAA
+    @media (max-width: 1023px)
```

Comment:
you can use +medium-screen() I think https://github.com/cozy/cozy-ui/blob/master/stylus/settings/breakpoints.styl#L13